### ISSUE TITLE
BCP intermittent error babelfish_view_def_pkey

### ISSUE THREAD

**Issue by staticlibs** [wiltondb/wiltondb#13]
BCP import error happens intermittently with the following messages in log:

```
2024-03-23 15:18:56.590 GMT [2424] ERROR:  duplicate key value violates unique constraint "babelfish_view_def_pkey"
2024-03-23 15:18:56.590 GMT [2424] DETAIL:  Key (dbid, schema_name, object_name)=(5, dbo, sp_describe_first_result_set_view_41) already exists.
2024-03-23 15:18:56.590 GMT [2424] CONTEXT:  SQL statement "CREATE VIEW sp_describe_first_result_set_view_41 as select * from adventureworks2019.Sales.SalesOrderDetail"
	PL/tsql function sp_describe_first_result_set(nvarchar,nvarchar,tinyint) line 3 at SQL statement
	SQL statement "EXEC sp_describe_first_result_set @p1"
	PL/tsql function inline_code_block line 1 at EXEC
```

Cannot reliably reproduce this, problem happens only on Windows.

**Comment by staticlibs** [comment]
The problem happens when BCP import is being initialized, it creates a view with name like `sp_describe_first_result_set_view_NNN`. This view is supposed to be dropped after the BCP import completes. But in some cases (unclear when exactly) it is not dropped completely. The following query can be used on Postgres port to clean up this view and allow BCP to work:

```
drop view if exists sp_describe_first_result_set_view_41;
delete from sys.babelfish_view_def where object_name = 'sp_describe_first_result_set_view_41';
```

**Comment by staticlibs** [comment]
This problem is fixed (or at least brought to the same behaviour as on Linux) by initializing the PRNG on Windows the same way it is initialized on Windows. Fix is included into [3.3-6-11-1 update](https://github.com/wiltondb/wiltondb/releases/tag/3.3-6-11-1).
